lib/routes/appstorrent/programs.ts

```diff
@@ -0,0 +1,105 @@
+import { Data, DataItem, Route } from '@/types';
+import got, { Options } from '@/utils/got';
+import { load } from 'cheerio';
+import { config } from '@/config';
+import cache from '@/utils/cache';
+import { getCurrentPath } from '@/utils/helpers';
+import * as path from 'node:path';
+import { art } from '@/utils/render';
+import { parseDate } from '@/utils/parse-date';
+import dayjs from 'dayjs';
+import { Context } from 'hono';
+const __dirname = getCurrentPath(import.meta.url);
+
+export const route: Route = {
+    path: '/programs',
+    categories: ['other'],
+    example: '/programs',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: 'AppsTorrent Programs',
+    maintainers: ['xzzpig'],
+    handler,
+    url: 'appstorrent.ru/programs/',
+    description: ``,
+};
+
+async function handler(ctx?: Context): Promise<Data> {
+    const limit = ctx?.req.query('limit') ? Number.parseInt(ctx?.req.query('limit') ?? '20') : 20;
+    const baseUrl = 'https://appstorrent.ru';
+    const currentUrl = `${baseUrl}/programs/`;
+    const gotOptions: Options = {
+        http2: true,
+        headers: {
+            'user-agent': config.ua,
+        },
```

Comment:
```suggestion
```
got will use the UA from config by default, no need to specify it again.
https://github.com/DIYgod/RSSHub/blob/1de5789a90ee5ff5dfab7ab7329f3ce1c28b06d8/lib/utils/got.ts#L69-L71